src/main/java/com/google/api/codegen/util/CommonAcronyms.java

```diff
@@ -30,6 +30,7 @@
           .put("HTTP", "Http")
           .put("XML", "Xml")
           .put("API", "Api")
+          .put("TEST", "Test")
```

Comment:
Hm, would rather not modify the real class just to add a test acronym. Maybe just call the service `NoTemplatesAPIService` instead?